uiComponents/button/buttonLink.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import constants from '../../constants';
+
+const LinkAsAButton = ({
+  href,
+  alt,
+  rel,
+  target,
+  disabled,
+  children,
+  large,
+}) => (
+  <a
+    rel={rel}
+    target={target}
+    className={`dont-apply-link-style button-link ${alt ? 'alt' : 'default'} ${
+      disabled ? 'disabled' : ''
+    }
+  ${large ? 'large' : 'small'}
+  `}
+    href={href}
+  >
+    <div className="content-wrapper">{children}</div>
+  </a>
+);
+
+const ClassicButton = ({ type, onClick, alt, large, disabled, children }) => (
+  <button
+    onClick={onClick}
+    type={type}
+    className={`dont-apply-link-style button-link ${alt ? 'alt' : 'default'} ${
+      disabled ? 'disabled' : ''
+    }
+  ${large ? 'large' : 'small'}
+  `}
+  >
+    <div className="content-wrapper">{children}</div>
+  </button>
+);
+
+const ButtonLink = ({
+  href,
+  alt,
+  rel,
+  target,
+  disabled,
+  children,
+  large,
+  type,
+  onClick,
+}) => {
+  if (!(onClick || type) && !href) {
+    throw new Error('Invalid props');
```

Comment:
on pourrait avoir un meilleur message d'erreur : 
"Invalid props. To make a link specify href, or to make a button specify onClick or type"

Reply:
Merci. Je corrige